styles/components/component.css

```diff
@@ -0,0 +1,35 @@
+.component-listing{
+    display:flex;
+    flex-direction:column;
+    padding:1rem;
+    /* background-color: grey; */
+    margin-top: 1.5rem;
+    width:100%;
+}
+
+.component-section{
+    width:100%;
+    border-bottom:solid 3px var(--clr-grey);
+    padding:0.25rem;
+}
+
+.component-container{
+    display:flex;
+    flex-direction: column;
+}
+
+.component-variant-container{
+    margin:1rem 0;
+}
+
+.component-variant-heading{
+    margin:1rem 0;
+}
+.component-variant-content{
+    padding:1rem;
+    display:flex;

```

Comment:
As flex is used many a times in your code. You can make this a utility class and use wherever required.

Reply:
Sure will definitely do that 👍🏽